lib/ui/src/modules/ui/components/left_panel/stories/tree_decorators.js

```diff
@@ -0,0 +1,76 @@
+import reactTreebeard from 'storybook-react-treebeard';
```

Comment:
Yep.. But it didn't work (they changed the export of the index file in the PR) But I am changing it in the forked version to the named export and it should fix this.